File: copy_images.js
import fs from 'fs';
import path from 'path';

const brainDir = 'C:/Users/mohma/.gemini/antigravity-ide/brain/385e7720-2ad7-4879-ac3f-59e8decc6561';
const publicImgDir = 'c:/Users/mohma/OneDrive/Desktop/stitch_lamsat_emaar_renovation_decor/public/images';

if (!fs.existsSync(publicImgDir)) {
  fs.mkdirSync(publicImgDir, { recursive: true });
}

const files = fs.readdirSync(brainDir);
files.forEach(file => {
  if (file.endsWith('.jpg') || file.endsWith('.png')) {
    const src = path.join(brainDir, file);
    const dest = path.join(publicImgDir, file);
    fs.copyFileSync(src, dest);
    console.log(`Copied ${file} to public/images/`);
  }
});

File: copy_images.py
import os
import shutil
import glob

brain_dir = r'C:\Users\mohma\.gemini\antigravity-ide\brain\385e7720-2ad7-4879-ac3f-59e8decc6561'
public_img_dir = r'c:\Users\mohma\OneDrive\Desktop\stitch_lamsat_emaar_renovation_decor\public\images'

os.makedirs(public_img_dir, exist_ok=True)

for jpg in glob.glob(os.path.join(brain_dir, '*.jpg')):
    filename = os.path.basename(jpg)
    dest = os.path.join(public_img_dir, filename)
    shutil.copy(jpg, dest)
    print(f"Copied {filename} to public/images/")

print("Finished copying images.")
